tests/model/component_import.cpp

```diff
@@ -201,14 +188,14 @@ TEST(ComponentImport, multipleImportAndParse)
 
     libcellml::PrinterPtr printer = libcellml::Printer::create();
     std::string a = printer->printModel(m);
-    EXPECT_TRUE((e1 == a) || (e2 == a));
+    EXPECT_EQ(e, a);
 
     // Parse
     libcellml::ParserPtr parser = libcellml::Parser::create();
-    libcellml::ModelPtr model = parser->parseModel(e2);
+    libcellml::ModelPtr model = parser->parseModel(e);
     EXPECT_EQ(size_t(3), model->componentCount());
     a = printer->printModel(model);
-    EXPECT_TRUE((e1 == a) || (e2 == a));
+    EXPECT_EQ(e, a);
```

Comment:
This is correct, as you get back the same model as parsed, but this is a different model to the one programmatically constructed above...

Reply:
Done in fca5344